index.bs

```diff
@@ -485,12 +429,6 @@ spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180
   execute correctly, even if the HTTPS version of a resource differs from the HTTP
   version.
 
-  Note: This algorithm returns `false` if the response is not <a
```

Comment:
Can we leave a note in saying that fetch ~~is doing an eligibility check~~ needs the typical opt-in via CORS headers or SRI won't work for cross-origin resources?